src/app/modules/shared/components/search-panel/search-panel.component.html

```diff
@@ -1,4 +1,4 @@
-<div>
-  <input type="text">
-  <button>Search</button>
-</div>
+<form class="search-form">
```

Comment:
Form based on one input. I think it's overhead.
https://www.uxbooth.com/articles/best-practices-for-search/ about design search field

Reply:
tag form was replaced with div, search button moved from the input field in 5be0a87